pmm-app/src/pmm-settings/Parts/AlertManager/AlertManager.tsx

```diff
@@ -77,12 +73,7 @@ const AlertManager = props => {
                   />
                 }
                 element={
-                  <InputField
-                    form={form}
-                    name="alert_manager_url"
-                    placeholder="Enter URL"
-                    style={{ width: '100%' }}
-                  />
+                  <InputField name="alert_manager_url" placeholder="Enter URL" style={{ width: '100%' }} />
```

Comment:
Better to use emotion for styling. Also, we should prefer classes over inline styles.